src/main/resources/application.yaml

```diff
@@ -140,3 +140,9 @@ proxy:
 
 reconciliation:
   api-key: ${RECONCILIATION_API_KEY}
+
+idam:
+  s2s-auth:
```

Comment:
why under 'idam'?

Reply:
`ServiceAuthorisationApi` is expecting the config under idam. `idam.s2s-auth.url`